database/queries/delete_pocky_user.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM pockyusers
```

Comment:
do we actually want to hard delete users? Or just mark as deleted?

Reply:
The users table is designed to be very light. Hard deleting them will have next-to-no effect, and they will be automatically added back in if they give or receive a peg.